monkey/monkey_island/cc/testing/environment/server_config_data_dir.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Name for this file doesn't make sense. If this is our new format of `server_config.json` then we should move this file to `test_common` folder (so we can reuse it for all tests that involve `server_config.json`) and rename it to `server_config_test_data.json` or something similar.

Reply:
I renamed it to `server_config_with_data_dir.json` to help alleviate some ambiguity. As for it's placement, it's currently in the `testing` directory. It looks like we have `testing` and `test_common`. Since we're planning on moving all of our tests under a common directory, we can sort out the finer details of where this file lives when we undertake that endeavor.